routes/campaign.php

```diff
@@ -268,6 +268,7 @@
     'locations' => 'LocationController',
     //'locations.map_points' => 'LocationMapPointController',
     'families' => 'FamilyController',
+    'families.character_family' => 'CharacterFamilyController',
```

Comment:
The danger of doing it in `Resources` is that laravel is adding `index`, `show`, `get`,  `create`, `store`,  `edit`, 'update` and `delete` routes automatically. We only need two routes, so just add those two individually instead, thanks